### ISSUE TITLE
How draw image to scale with window size

### ISSUE THREAD

**Issue by jakeklassen** [kmamal/node-sdl#28]
First off thanks for this lib, super cool 😎 

I'm trying to render a 128x128 pixel canvas to a 512x512 window. Based on the [window.render()](https://github.com/kmamal/node-sdl#windowrenderwidth-height-stride-format-buffer) docs I noted **The provided image will be stretched over the entire window.**. I'm not sure how this _stretching_ works. Is it automatic?

Here's some sample code:

```ts
import sdl from "@kmamal/sdl";
import Canvas, { loadImage } from "canvas";

// 8x8 image
const ship = await loadImage("assets/player-ship.png");

const gameWidth = 128;
const gameHeight = 128;
const scale = 4;

const window = sdl.video.createWindow({
  title: "Canvas",
  width: gameWidth * scale,
  height: gameHeight * scale,
  vsync: true,
});

const { width, height } = window;
const canvas = Canvas.createCanvas(gameWidth, gameHeight);

const ctx = canvas.getContext("2d");
ctx.imageSmoothingEnabled = false;

ctx.fillStyle = "black";
ctx.fillRect(0, 0, gameWidth, gameHeight);
ctx.drawImage(
  ship,
  canvas.width / 2 - ship.width / 2,
  canvas.height / 2 - ship.height / 2,
);

const buffer = canvas.toBuffer("raw");

// Is this correct??
window.render(gameWidth, gameHeight, gameWidth * 4, "bgra32", buffer);
```

Here is the resulting window output

![image](https://user-images.githubusercontent.com/1383068/229809729-ad1afc07-49ea-4386-98ad-84737afa626d.png)

It's drawing the 128x128 buffer unscaled in the top left.

Any help is appreciated. Maybe this doesn't work the way I thought, and I need to scale the `canvas` before drawing the buffer?

**Comment by kmamal** [comment]
Thanks for the feedback! That's absolutely a bug. I thought I had fixed this but apparently I didn't. Anyway it should be fixed now in `v0.6.9`. Can you have another look?

**Comment by jakeklassen** [comment]
Looks good now! Cheers 🍻
